stablehlo/tests/stablehlo_legalize_to_vhlo.0_10_0.mlir

```diff
@@ -1211,7 +1211,7 @@ func.func @op_dynamic_conv(%arg0: tensor<1x8x8x207xf32>, %arg1: tensor<3x3x207x1
     padding = dense<1> : tensor<2x2xi64>,
     lhs_dilation = dense<2> : tensor<2xi64>,
     rhs_dilation = dense<2> : tensor<2xi64>,
-    window_reversal = dense<true> : tensor<2xi1>,
+    window_reversal = array<i1: true, true>,
```

Comment:
Where did you land on the round-trip issue? Should we keep these as dense for now?

Reply:
For this attr IMO it's not worth going through an intermediate "array or elements" step since it's the only attr that's a 1D range of booleans. Also since it doesn't seem like it's used a whole lot IMO it's okay to directly port it to array. We can definitely port it in a later change though, WDYT?